x-pack/plugins/reporting/server/plugin.ts

```diff
@@ -92,19 +87,11 @@ export class ReportingPlugin
       runValidations(config, elasticsearch, browserDriverFactory, this.logger);
 
       this.logger.debug('Start complete');
-      this.start$.next(true);
     })().catch((e) => {
       this.logger.error(`Error in Reporting start, reporting may not function properly`);
       this.logger.error(e);
     });
 
     return {};
   }
-
-  public getReportingCore() {
```

Comment:
The `createMockReportingCore` function has been refactored, and this is no longer needed.